src/ai/mod.rs

```diff
@@ -20,23 +14,46 @@ pub const AVAILABLE_AI: &[&str; 1] = &["OpenAI"];
 
 pub async fn generate_message(
     configuration: super::Configuration,
-    ticket: Option<Ticket>,
+    tickets: Vec<Ticket>,
```

Comment:
Now for the ticket we are receiving a vector so the user can pass nothing, one or more tickets